### DISCUSSION TITLE
Help with installing on RunCloud managed server

### DISCUSSION THREAD

**Discussion by bartrylant** [plausible/analytics#4501] [Self-Hosted Support]
Hi

So I followed this guide (https://www.digitalocean.com/community/tutorials/how-to-install-plausible-analytics-on-ubuntu-22-04) until the end of Step 1. However, RunCloud has a custom nginx version and I can't "connect" the docker image with Plausible with the nginx server. Any help here?

**Comment by ruslandoga** [comment]
👋 @bartrylant 

What is the problem exactly?

**Reply by ruslandoga** [reply]
That guide seems quite outdated, I would recommend https://github.com/plausible/community-edition instead :) It uses [Docker Compose](https://docs.docker.com/compose/) and [Caddy,](https://caddyserver.com) by default.
